app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,28 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
```

Comment:
||= có tác dụng gì em

Reply:
chưa trả lời này e

Reply:
ở cmt trên em có trả lời ruifa a, toán tử đó có tác dụng kiểm tra xem có current_user hay không, nếu không có thì giá trị của current_user = vế sau dấu bằng